bpf/lib/common.h

```diff
@@ -239,8 +239,6 @@ __revalidate_data_pull(struct __ctx_buff *ctx, void **data, void **data_end,
 
 /* Structure representing an IPv4 or IPv6 address, being used for:
  *  - key as endpoints map
- *  - key for tunnel endpoint map
- *  - value for tunnel endpoint map
```

Comment:
Nit: You're leaving a list of one item, could maybe be a normal sentence instead.